src/com/magento/idea/magento2plugin/util/magento/MagentoVersion.java

```diff
@@ -70,4 +70,26 @@ public static String get(final Project project, final String magentoPath) {
     private static String getFilePath(final String magentoPath) {
         return magentoPath + File.separator + ComposerJson.FILE_NAME;
     }
+
+    /**
+     * compare Magento Version.
+     *
+     * @param str1 String
```

Comment:
Please use self-explanatory variable names. `str1 ` doesn’t say anything to us. Also, please describe a few words about how this method works. It about to be commonly used. Thank you!